src/gui/tray/activitylistmodel.cpp

```diff
@@ -65,11 +69,14 @@ QHash<int, QByteArray> ActivityListModel::roleNames() const
     roles[ActionIconRole] = "icon";
     roles[ActionTextRole] = "subject";
     roles[ActionsLinksRole] = "links";
+    roles[ActionsLinksContextMenuRole] = "linksContextMenu";
```

Comment:
can you also modify the first line like this ?
`QHash<int, QByteArray> roles;`
becomes
`auto roles = QAbstractListModel::roleNames();`